README.md

```diff
@@ -1,17 +1,15 @@
 # nRF IEEE 802.15.4 radio driver
 
-[![Build Status][travis-svg]][travis]
-
-[travis]: https://travis-ci.org/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver
-[travis-svg]: https://travis-ci.org/NordicSemiconductor/nRF-IEEE-802.15.4-radio-driver.svg?branch=master
-
-The nRF IEEE 802.15.4 radio driver implements the IEEE 802.15.4 PHY layer on the Nordic Semiconductor nRF52811 and nRF52840 SoCs.
+The nRF IEEE 802.15.4 radio driver implements the IEEE 802.15.4 PHY layer on the following Nordic Semiconductor SoCs:
+    - nRF52811

```

Comment:
It seems that this enumeration renders as a single line. Is that intentional?

Reply:
No, but it's tagged like this in the source repository. I would keep it and fix later